lib/vim-flavor/shellutility.rb

```diff
@@ -2,11 +2,14 @@ module Vim
   module Flavor
     module ShellUtility
       def sh script
-        output = send(:`, script)
-        if $? == 0
-          output
-        else
-          raise RuntimeError, output
+        IO.popen(['bash', '-c', script], 'r') do |io|
+          output = io.read
+
+          if $?.success?
```

Comment:
Now many tests are failed by this line.  It seems to be wrong to use `$?` inside a block given to `IO::popen`.  According to http://ruby-doc.org/core-2.3.0/IO.html#method-c-popen:

> If a block is given, Ruby will run the command as a child connected to Ruby with a pipe. Ruby’s end of the pipe will be passed as a parameter to the block. **At the end of block, Ruby closes the pipe and sets $?.** In this case IO.popen returns the value of the block.